variation/__init__.py

```diff
@@ -23,6 +23,17 @@
 logger.setLevel(logging.DEBUG)
 logger.handlers = []
 
+logging.getLogger("boto3").setLevel(logging.INFO)
+logging.getLogger("botocore").setLevel(logging.INFO)
+logging.getLogger("urllib3").setLevel(logging.INFO)
+logging.getLogger("python_jsonschema_objects").setLevel(logging.INFO)
+logging.getLogger("hgvs.parser").setLevel(logging.INFO)
+logging.getLogger("biocommons.seqrepo.seqaliasdb.seqaliasdb").setLevel(logging.INFO)
+logging.getLogger("biocommons.seqrepo.fastadir.fastadir").setLevel(logging.INFO)
+
+# logging.getLogger("bioregistry.resource_manager").setLevel(logging.INFO)
```

Comment:
You probably don't need this, right?

Reply:
Lol was hoping you'd miss that part. No I don't. I just didn't want to make another commit, but will remove it.

Reply:
FWIW you can just squash commits on merge ("squash" is a v satisfying word)